### PR TITLE
add l2 rewrite function for convenient access

### PR DESCRIPTION
**Pull request by toanju**

### CODE REVIEW

File: include/rofl/ofdpa/rofl_ofdpa_fm_driver.hpp [hunk 1]
```diff
@@ -111,6 +111,32 @@ class rofl_ofdpa_fm_driver final {
...
+    rofl::openflow::cofactions write_actions(dpt.get_version());
+    write_actions.set_action_group(rofl::cindex(0)).set_group_id(group_id);
+
+    enable_policy_acl_ipv4_vlan(dpt, match, false, 0, 0,
+                                rofl::openflow::cofactions(), write_actions);
+  }
+
+  void disable_send_to_l2_rewrite(rofl::crofdpt &dpt, uint16_t vid,
+                                  const rofl::caddress_ll &dst,
+                                  uint32_t group_id) {
```
Comment (octopusx) [inline include/rofl/ofdpa/rofl_ofdpa_fm_driver.hpp]: group_id isn't used anywhere. Remove?

Comment (toanju) [inline include/rofl/ofdpa/rofl_ofdpa_fm_driver.hpp]: write_actions.set_action_group(rofl::cindex(0)).set_group_id(group_id);

Comment (octopusx) [inline include/rofl/ofdpa/rofl_ofdpa_fm_driver.hpp]: Wrong line, wrong function :)

Comment (toanju) [inline include/rofl/ofdpa/rofl_ofdpa_fm_driver.hpp]: sorry, always getting confused with GH in this case... will fix it :)

**Pull request merged**